EMacco/GoBanka
# #168913614 Show Full Receipt

#### What does this PR do?
This PR allows a user to be able to view the receipt of a particular transaction

#### Description of Task to be completed?
Display the full information for the user's transactions

#### How should this be manually tested?
- Checkout to this branch
- Login using email and password
- Perform a transaction
- Click on the transaction that was just performed
- You should see the full information about that particular transaction

#### What are the relevant pivotal tracker stories?
[#168913614](https://www.pivotaltracker.com/story/show/168913614)

#### Screenshots (if appropriate)
